x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -858,6 +858,31 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                     )
                     .privileges("create_index", "delete_index", "read", "index", IndicesAliasesAction.NAME, UpdateSettingsAction.NAME)
                     .build(),
+                // For destination indices of the Threat Intel (ti*) packages that ships a transform for supporting IOC expiration
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(".logs-ti_*latest*")
```

Comment:
```suggestion
                    .indices(".logs-ti_*_latest.*")
```

I'm trying to be as specific as possible.

Reply:
Updated as suggested